packages/components/pie-modal/test/visual/pie-modal.spec.ts

```diff
@@ -88,3 +91,45 @@ sizes.forEach((size) => {
         await percySnapshot(page, `Modal - size = ${size}`);
     });
 });
+
+test('Should render with full width when size = medium and isFullWidthBelowMid = true', async ({ page, mount }) => {
+    await mount(PieModal, {
+        props: {
+            heading: 'This is a modal heading',
+            headingLevel: 'h2',
+            isOpen: true,
+            size: 'medium',
+            isFullWidthBelowMid: true,
+        },
+    });
+
+    await percySnapshot(page, 'Modal - size = medium & isFullWidthBelowMid = true');
+});
+
+test('Should not render with full width when size is NOT medium and isFullWidthBelowMid = true', async ({ page, mount }) => {
```

Comment:
What is the viewport width for these tests? Large modals should also be full width below the breakpoint regardless of the value of `isFullWidthBelowMid`.

Reply:
Actually these are visual tests so the viewport varies, perhaps we should use `small` for these?

Reply:
yeah I wasn't sure if there was a way I could only specify a particular viewport in the test. I'll change to small - that makes more sense 👍

Reply:
done 😄 thank you!